src/content/peerconnection/munge-sdp/js/test.js

```diff
@@ -25,11 +25,18 @@ test('Munge SDP sample', function(t) {
     return driver.findElement(webdriver.By.id('getMedia')).click();
   })
   .then(function() {
+    return driver.wait(function() {
+      return driver.executeScript('return typeof window.localStream ' +
+        '!== \'undefined\'');
+    });
+  })
+  .then(function() {
     t.pass('got media');
     return driver.findElement(webdriver.By.id('createPeerConnection')).click();
   })
   .then(function() {
-    return driver.findElement(webdriver.By.id('createOffer')).click();
+    return driver.wait(webdriver.until.elementIsVisible(
```

Comment:
Did this on purpose, i.e. want to see which is more stable, isEnabled or elementIsVisible.